src/main/app/common/court-calculations/courtDecision.ts

```diff
@@ -0,0 +1,38 @@
+import { Moment } from 'moment'
+
+export enum DecisionType {
+  CLAIMANT_IN_FAVOUR_OF_DEFENDANT = 'CLAIMANT_IN_FAVOUR_OF_DEFENDANT',
+  CLAIMANT = 'CLAIMANT',
+  DEFENDANT = 'DEFENDANT',
+  COURT = 'COURT'
```

Comment:
as we were discussing earlier, this replicates backend model. It will be nice to change both at some point but not related to this PR as i just changed file name here.